sadl3/com.ge.research.sadl.parent/com.ge.research.sadl.jena-wrapper-for-sadl/src/main/java/com/ge/research/sadl/jena/reasoner/builtin/ITypedBaseBuiltin.java

```diff
@@ -56,6 +56,13 @@ public interface ITypedBaseBuiltin {
 	 */
 	abstract boolean canProcessGraphPatternArguments();
 	
+	/**
+	 * Method to answer the number of arguments (VariableNodes) expected
+	 * to have values bound to them as results of built-in evaluation.
+	 * @return
+	 */
+	abstract int numOutputArgs();
```

Comment:
I searched the PR's diffs for numOutputArgs and all of the implementations I saw always return 1.  Is there any use case in which an implementation might return another value such as 0 or 2?